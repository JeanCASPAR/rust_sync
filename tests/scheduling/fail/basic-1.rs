use rust_sync::sync;

sync! {
    #![pass(2)]

    node oui() = ()
    where
        a : int = a;
}

fn main() {}
