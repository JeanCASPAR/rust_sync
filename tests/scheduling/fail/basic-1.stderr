error: scheduling error: the following expressions form a cycle : a
 --> tests/scheduling/fail/basic-1.rs:8:19
  |
8 |         a : int = a;
  |                   ^
